pepper-apis/src/main/resources/org/broadinstitute/ddp/db/dao/PdfDao.sql.stg

```diff
@@ -103,10 +103,26 @@ select base.pdf_base_template_id    as template_id,
   left join pdf_mailing_address_template as taddr on taddr.pdf_base_template_id = base.pdf_base_template_id
   left join pdf_physician_institution_template as tphys on tphys.pdf_base_template_id = base.pdf_base_template_id
   left join institution_type as itype on itype.institution_type_id = tphys.institution_type_id
+>>
+bulkQueryBaseTemplatesByVersionId() ::= <<
+<selectBaseTemplates()>
  where pvt.pdf_document_version_id = :versionId
  order by pvt.template_order asc
 >>
 
+findBaseTemplatesByTemplateId() ::= <<
+<selectBaseTemplates()>
+where pvt.pdf_base_template_id = :templateId
+>>
+
+findTemplateIdsByVersionId() ::= <<
+    select pvt.pdf_base_template_id
+    from pdf_version_template as pvt
+    join pdf_base_template as base on base.pdf_base_template_id = pvt.pdf_base_template_id
```

Comment:
Doesn't look like we need the join to the `pdf_base_template` table.

Reply:
Right! Dropping join.
7bb503c8